scipy/spatial/tests/test_distance.py

```diff
@@ -957,13 +958,13 @@ def test_pdist_correlation_iris_nonC(self):
         Y_test2 = wpdist(X, 'test_correlation')
         _assert_within_tol(Y_test2, Y_right, eps)
 
-    @pytest.mark.parametrize("p", [1.0, 2.0, 3.2, np.inf])
+    @pytest.mark.parametrize("p", [0.1, 0.25, 1.0, 2.0, 3.2, np.inf])
     def test_pdist_minkowski_random_p(self, p):
-        eps = 1e-05
+        eps = 1e-13
```

Comment:
Forgot to ask. How did you select the tol values? Usually when we have coarse tolerances it's because on some platforms it would fail. (If could pass here and fail in the wheel repo jobs.)

Reply:
The tolerances were coarse due to https://github.com/scipy/scipy/issues/15058.

On relative terms, I consider 1e-13 to be quite loose (when eps is ~1e-16), i.e. three orders of magnitude of leeway.